File: msvc-match/src/main/java/com/project/msvc_match/config/SwaggerConfig.java
package com.project.msvc_match.config;

import io.swagger.v3.oas.models.OpenAPI;
import io.swagger.v3.oas.models.info.Info;
import org.springframework.context.annotation.Bean;
import org.springframework.context.annotation.Configuration;

@Configuration
public class SwaggerConfig {

    @Bean
    public OpenAPI customOpenApi() {
        return new OpenAPI()
                .info(new Info()
                        .title("API Juegos")
                        .version("1.0")
                        .description("Documentación OpenAPI del microservicio de gestión de partidas de eSports Arena Manager"));
    }
}

File: msvc-sanction/src/main/java/com/project/msvc_sanction/config/SwaggerConfig.kt
package com.project.msvc_sanction.config

import io.swagger.v3.oas.models.OpenAPI
import io.swagger.v3.oas.models.info.Info
import org.springframework.context.annotation.Bean
import org.springframework.context.annotation.Configuration


@Configuration
class SwaggerConfig {
    @Bean
    fun customOpenAPI(): OpenAPI? {
        return OpenAPI()
            .info(
                Info()
                    .title("eSports Arena Manager API")
                    .version("1.0")
                    .description("Documentación OpenAPI del microservicio de gestión de sanciones de eSports Arena Manager")
            )
    }
}